src/utils/CAPTCHADetector.ts

```
@@ -0,0 +1,1163 @@
+
+        // Handle patterns like: 1753230880859_0001.png
+        const match = filename.match(/^(\d+)_\d+\.png$/)
+        if (match) {
+            const timestamp = parseInt(match[1], 10)
+            console.log(
+                `🔍 [CAPTCHADetector]   - Pattern 1 match: ${timestamp}`,
+            )
+            return timestamp
+        }
+
+        // Fallback: try to extract any number sequence
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)